src/chain.h

```diff
@@ -378,6 +378,7 @@ class CChain {
 
     /** Efficiently check whether a block is present in this chain. */
     bool Contains(const CBlockIndex *pindex) const {
+        if (!pindex) return false;  // null pointer isn't in this chain
```

Comment:
That's a bigger change because a lot of callers would have to dereference first.